.pre-commit-config.yaml

```diff
@@ -10,4 +10,11 @@ repos:
   - repo: https://github.com/pre-commit/mirrors-autopep8
     rev: v2.0.4
     hooks:
-    - id: autopep8
\ No newline at end of file
+    - id: autopep8
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+    - id: check-toml
+    - id: forbid-new-submodules
+    - id: end-of-file-fixer
+    - id: check-yaml
```

Comment:
These changes were made and applied to some files - I can undo this, but I think these checks are reasonable

Reply:
Seems reasonable to me